app/routes/events/components/EventEditor/index.tsx

```diff
@@ -594,13 +604,45 @@ const EventEditor = () => {
                     />
                   </div>
                 )}
-                <Field
-                  label="Påmeldingstype"
-                  name="eventStatusType"
-                  component={SelectInput.Field}
-                  fieldClassName={styles.metaField}
-                  options={eventStatusTypes}
-                />
+                {spyValues((values: EditingEvent) => {
+                  // Adding an initial pool if the event status type allows for it and there are no current pools
+                  if (
+                    ['NORMAL', 'INFINITE'].includes(
+                      values.eventStatusType?.value
+                    )
+                  ) {
+                    if (values.pools.length === 0) {
+                      values.pools = [
+                        {
+                          name: 'Pool #1',
+                          registrations: [],
+                          activationDate: moment(values.startTime)
+                            .subtract(7, 'd')
+                            .hour(12)
+                            .minute(0)
+                            .toISOString(),
+                          permissionGroups: [],
+                        },
+                      ];
+                    }
+                  } else {
+                    // Removing all pools so that they are not validated on submit
+                    if (values.pools.length > 0) {
+                      values.pools = [];
+                    }
+                  }
+
+                  return (
```

Comment:
Stonks

Reply:
haha yes the code here is awful. I'm oblivious as to why this magically worked before, and I'd like to refactor it - but I'm not going to spend more hours (read 5) on this now.